### ISSUE TITLE
ios tracking fired only first time then stops while it works fine on simulator

### ISSUE THREAD

**Issue by oshek14** [transistorsoft/cordova-background-geolocation-lt#582]
## Your Environment
* Plugin version:
* Platform: iOS 
* OS version:10.2.1
* Device manufacturer / model:
* Cordova version 7.2
* Cordova platform version (`cordova platform ls`):
* Plugin config

## Expected Behavior
i have distance filter set to 10 meters. it works on xcode simulator. i set on simulator freeway drive which means simulator is presumed to be moving. and every ten meters passed,it updates location and puts in database. all is working fine but on my ios 10.2.1 version iphone 5, it only puts first coordinates as soon as app gets opened and stops getting locations every ten meters. 

why does that happen? if it works on simulator,it should work on mobile too.
## Steps to Reproduce

1.
2.
3.
4.

**Comment by christocracy** [comment]
This is normal.  Read the [Philosophy of Operation](https://github.com/transistorsoft/cordova-background-geolocation-lt/wiki/Philosophy-of-Operation)

tl;dr  You have to move *at least* 200 meters beyond the "stationary location" before tracking engages.

**Comment by oshek14** [comment]
i moved even 1000 meters 👍 but with no luck it stil doesn't work

**Comment by christocracy** [comment]
Show me plugin's logs.  See [Debugging](https://github.com/transistorsoft/cordova-background-geolocation-lt/wiki/Debugging) wiki.
